src/components/FeatureViews/Accounts/TabComponents/InstructorCourses.js

```diff
@@ -8,35 +8,53 @@ import Paper from "@material-ui/core/Paper";
 import Typography from "@material-ui/core/Typography";
 import UnconfirmIcon from "@material-ui/icons/Cancel";
 import Moment from "react-moment";
+import moment from "moment";
 
 import gql from "graphql-tag";
 import { useQuery } from "@apollo/react-hooks";
+import Loading from "components/OmouComponents/Loading";
+import { DayAbbreviation } from "utils";
 
 export const GET_INSTRUCTOR_ENROLLMENTS = gql`
   query InstructorEnrollments($instructorID: ID!) { 
     courses(instructorId: $instructorID) {
       id
-      dayOfWeek
       description
       startDate
-      startTime
       endDate
-      endTime
       isConfirmed
       title
+      availabilityList {
+        id
+        dayOfWeek
+        endTime
+        startTime
+      }
     }
   }
 `;
 
+const checkTimes = (courseTimes) => {
+  let start = courseTimes[0].startTime;
+  let end = courseTimes[0].endTime;
+
+  for (let course of courseTimes) {
+    if (course.startTime !== start || course.endTime !== end) {
+      return false;
+    }
+  }
+  return true;
+}
+
 const InstructorCourses = ({ instructorID }) => {
 
   const { loading, error, data } = useQuery(GET_INSTRUCTOR_ENROLLMENTS, {
 		variables: {instructorID}
 	})
 
-	if (loading ) return null;
+	if (loading ) return <Loading small/>;
 
-  if (error ) return "Unable to load courses";
+  if ( error ) return console.error(error.message);
```

Comment:
there should be an error component you can use

Reply:
I looked around and I could not find one. I saw in multiple other components doing this or` <Typography>There was an error {message.error}</Typography>`. Where can I find the <Error/>, is that part of Material UI?

Reply:
hm... i thought Jeffrey made one. it's ok. but yes, you do want to display this on the component itself

Reply:
Ok I do not see an Error component inside of OmouComponents, so I will do  `<Typography>There was an error {message.error}</Typography>` instead.